esp/services/ws_configmgr/ws_configmgrService.cpp

```diff
@@ -593,6 +596,103 @@ bool Cws_configMgrEx::onFetchNodes(IEspContext &context, IEspFetchNodesRequest &
 }
 
 
+bool Cws_configMgrEx::onGetNodeCopy(IEspContext &context, IEspGetNodeCopyRequest &req, IEspGetNodeCopyResponse &resp)
+{
+    std::string nodeId = req.getNodeId();
+    std::string sessionId = req.getSessionId();
+    Status status;
+
+    ConfigMgrSession *pSession = getConfigSession(sessionId, true);
```

Comment:
Do we need a nullptr check on pSession? It looks like sessionId is coming from the user

Reply:
No. The getConfigSession method will throw an exception if the ID is not valid.